exec/tests/unit/commands/test_run.py

```diff
@@ -203,7 +203,6 @@ def test_get_image_tag(image, tag, expected_image):
     assert expected_image == actual_image
 
 
-@pytest.mark.parametrize("streaming", (True, False))
```

Comment:
Curious, why was this parametrization removed?

Reply:
I went a bit too quickly, I think - I wanted this to only test cases that didn't raise and in doing so forgot that verification of the settings in `klio-job.yaml` needed to test both streaming and batch. 

I've added it back + a test case that checks for the raised exception.